features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/components/RequestVerificationHeader.kt

```diff
@@ -50,9 +50,9 @@ internal fun RequestVerificationHeader(
     onDismissClicked: () -> Unit,
     modifier: Modifier = Modifier,
 ) {
-    Box(modifier = Modifier.padding(horizontal = 16.dp, vertical = 8.dp)) {
+    Box(modifier = modifier.padding(horizontal = 16.dp, vertical = 8.dp)) {
         Surface(
-            modifier.fillMaxWidth(),
+            Modifier.fillMaxWidth(),
```

Comment:
Note to myself: I will double check that the banner is still displayed.

Reply:
OK, it's displayed, but I have to go back to the home screen and restart the app after a login to see the banner, and it's the same to see the "Complete verification" item in the settings, so this is not something new introduced by this change.